src/test/kotlin/org/jetbrains/research/transformations/maintests/SetupParserTest.kt

```diff
@@ -0,0 +1,7 @@
+package org.jetbrains.research.transformations.maintests
+
+import org.jetbrains.research.transformations.SetupParser
+
+fun main() {
+    SetupParser.checkSetup()
+}
```

Comment:
It's nice to have an example of how setup parser, but I don't see any reasons to make it in this way. Why do you need a main function in test? When do you run it?